Makefile

```
@@ -0,0 +1,45 @@
+	uv run ruff check --fix
+
+format:
+	uv run ruff format
+
+typecheck:
+	uv run ty check
+
+deadcode:
+	uv run vulture --min-confidence 100 --exclude ".venv" .
+
+rebuild:
```

checkmake (0.2.2)

[warning] Target body for "rebuild" exceeds allowed length of 5 (13).

(maxbodylength)